src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,56 @@
+package duke;
+
+import com.sun.source.util.TaskListener;
+import duke.Command.Command;
+import duke.task.Task;
+import javafx.application.Application;
+
+import java.io.IOException;
+import java.text.ParseException;
+
+import javafx.application.Application;
+import javafx.scene.Scene;
+import javafx.scene.control.Label;
+import javafx.stage.Stage;
+
+/**
+ * The Duke class is the main class and drives the program
+ */
+
+public class Duke{
+
+    private duke.Storage storage;
+    private TaskList tasks;
+    private Ui ui;
+
+    /**
+     * Constructor for a new driver system
+//     * @param filePath  the file path of the schedule list
```

Comment:
I like that you did a javadoc comment, but did you accidentally leave a // behind?

![image](https://user-images.githubusercontent.com/25708498/92355418-c1ba2d80-f116-11ea-90c6-1d60f17b7b5d.png)